### DISCUSSION TITLE
HELP ME PLEASE - I forget my email

### DISCUSSION THREAD

**Discussion by Kremilly** [community/community#22896] [New to GitHub]
<p>Hi! Community</p>
<p>So I need help to recover an old account of my team with the username: <strong>rilexsoft</strong> , the problem is that I forget the email that use for register on GitHub (this account was created between 2014/2015. And want this account for reactive my team and create new open-source projects.</p>
<p>If someone can help me, I gonna happy.</p>
<p>(Sorry my english, this language isn’t my first language)</p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/thesilvacesar">@thesilvacesar</a>,</p>
<p>Since this is an account-specific question, I’d recommend reaching out to our private support team via our contact form <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>, so that we can better assist you.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>

**Comment by Kremilly** [comment]
<p>I got try this, but not possible recover my account…</p>
<p>Thank you so much !</p>
